packages/main/src/themes/FormItem.css

```diff
@@ -57,13 +60,9 @@
 	box-sizing: border-box;
 }
 
-::slotted([ui5-input]) {
+::slotted(*) {
```

Comment:
Let's exclude the [ui5-checkbox] until its focus appearance is changed, otherwise it will look as a bug - short text but focus surrounding large empty space